src/main/java/uk/gov/indexer/ctserver/AuditProofs.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.indexer.ctserver;
+
+import java.util.List;
+
+public class AuditProofs {
```

Comment:
nitpick: I think this is a single AuditProof, not a collection of many AuditProofs